### ISSUE TITLE
Widescreen

### ISSUE THREAD

**Issue by jrtberlin** [ionelmc/python-darkslide#11]
Is there an easy way to get slides in 16:9 aspect ratio instead of 4:3?

**Comment by ionelmc** [comment]
Should be possible by messing with the widths in css. I don't have a copy-paste solution for you, you have to look at the current styling and try stuff.

**Comment by jrtberlin** [comment]
Thanks for your reply. CSS was my first guess, I just wanted to know if there is already a baked-in solution.

**Comment by epmoyer** [comment]
@jrtberlin: It's been a year and a half, but I needed wide slides last week so I ended up figuring it out :)  PR above.

@ionelmc: You were correct; the architecture is pretty well thought out and I was able to achieve 16x9 via css edits.  If css were a better "language" (i.e. with the ability to do #defines and math) then this change have been a  "one liner" :).  All the changes followed logically from the width increase; the only trick was finding the 10 - 15 values impacted by the new width.
